Procfile.dev

```diff
@@ -1,5 +1,5 @@
 # vim: ft=yaml
-web: env RUBY_DEBUG_OPEN=true bin/rails server -p 3000
+web: env RUBY_DEBUG_OPEN=true bin/rails server -p 5000
```

Comment:
5000 is a prob for local dev as this is used by AirDrop on mac.

Reply:
Looks like it's used for AirPlay receiver. But yes I'll change it to 8080.

I changed it from 3000 so that you could run test/learning rails apps on the same machine.